DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/SRSnapshotTests.swift

```diff
@@ -113,7 +113,32 @@ final class SRSnapshotTests: SnapshotTestCase {
     }
 
     func testImages() throws {
-        try takeSnapshotFor(.images, with: [.allow, .mask], shouldRecord: shouldRecord, folderPath: snapshotsFolderPath)
+        try takeSnapshotFor(
+            .images,
+            with: [.allow, .mask],
+            shouldRecord: shouldRecord,
+            folderPath: snapshotsFolderPath
+        )
+    }
+
+    func testImages_MaskAll() throws {
```

Comment:
There's a case for making it in similar fashion to privacy, but I think it's worth touching this once we have all the Fine Grained Masking / Masking Config RFCs approved.